src/fmu/dataio/datastructure/meta/content.py

```diff
@@ -197,9 +197,9 @@ class Content(BaseModel):
     is_prediction: bool = Field(
         title="Is prediction flag",
     )
-    layout: Optional[str] = Field(
+    layout: Optional[enums.Layout] = Field(
```

Comment:
This is what changed the schema. It gives a reference to the enum now rather than accepting a generic string.